app/views/alaveteli_pro/draft_info_request_batches/_draft_info_request_batch.html.erb

```diff
@@ -0,0 +1,39 @@
+<div id="draft-info-request-batch-<%= draft_info_request_batch.id %>"
+     class="request">
+  <h3 class="request__title">
+    <% if draft_info_request_batch.embargo_duration %>
+      <span class="embargo-indicator embargo-indicator--small">Private</span>
+    <% end %>
+    <span><%= _("Batch request") %></span>
```

Comment:
Hmm, I thought the link_to helper fell through to using the path as the link - that's what seems to happen for me:

![screen shot 2017-05-05 at 09 22 01](https://cloud.githubusercontent.com/assets/663700/25738335/6485eb46-3174-11e7-8635-b1f9f1ddd11e.png)

Does that not work for you, or do you think I should change it to something generic (like "no title") anyway?